engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/functional/CustomNodeProcessSpec.scala

```diff
@@ -246,7 +246,7 @@ class CustomNodeProcessSpec extends AnyFunSuite with Matchers with ProcessTestHe
 
     val thrown = the [IllegalArgumentException] thrownBy processInvoker.invokeWithSampleData(process, data)
 
-    thrown.getMessage shouldBe "Compilation errors: ExpressionParserCompilationError(Unresolved reference 'input',proc2,Some(all),#input.id)"
+    thrown.getMessage should startWith ("Compilation errors: ExpressionParserCompilationError(Unresolved reference 'input',proc2,Some(all),#input.id)")
```

Comment:
maybe we can assert with "shouldBe" to show that there are actually more then one error? Or have assertion for multiple errors in other test?

Reply:
I added a new test for that